pkg/envoy/ads/response.go

```diff
@@ -13,17 +11,48 @@ import (
 	"github.com/openservicemesh/osm/pkg/envoy"
 )
 
+const (
+	// fullUpdateStr is a constant string value to identify full XDS update times on metric labels
+	fullXDSUpdateStr = "FULL"
```

Comment:
Instead of full, makes more sense to change this to ADS just because A=aggregated XDS.
```suggestion
	ADSUpdateStr = "ADS"
```

Reply:
way better, +1